src/components/CardList/CardList.js

```diff
@@ -1,13 +1,17 @@
+import withLoadingDelay from "../../UI/withLoadingDelay/withLoadingDelay";
 import Card from "../Card/Card";
 import classes from "./CardList.module.css";
 
+const CardWithDelay = withLoadingDelay(Card);
+
 const CardList = (props) => {
   return (
     <div className={classes["card-list"]}>
       {props.data.map((cardData) => (
-        <Card
+        <CardWithDelay
           id={cardData.id}
           key={cardData.id}
+          styles={{margin: "100px 180px"}}
```

Comment:
I think its not a good idea to mix Loader and Card props.
May be it would be better to set Loader styles in HOC init function (above)?